apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Using specific templates in PDPs and PLPs'
+sidebar_label: 'Multiple page templates'
+---
+
+<header>
+
+# Multiple page templates
+
+</header>
+
+Multiple Page Template means that PDPs and PLPs, previously defined as singleton in the CMS, can now assume different page templates based on some rules. This approach is almost similar to the PDP and PLP, but they will be explained separately since each page type has particularities.
```

Comment:
> previously defined as singleton in the CMS

So, we won't have the option to create a singleton page anymore, right?
https://help.vtex.com/en/tutorial/managing-singleton-content-types--VBibMN1BqV2OFmavUFCdo?&utm_source=autocomplete

Reply:
This option exists from the point of view of CMS for other kind of pages. But does not make sense for PLPs and PDPs anymore.